### ISSUE TITLE
selector, trial 프로그램이 잡는 lock 질문

### ISSUE THREAD

**Issue by kjp4155** [hyojeonglee/osfall2019#41]
안녕하세요 조교님,

Project3 spec document에 보면 아래처럼 selector와 trial이 잡는 lock에 대해 설명되어 있습니다.

> selector: A program accepts a starting integer as the only argument. When running, first, your program must take the write lock for when the device is positioned at 0 <= degree <= 180.

> trial: A program accepts an integer identifier as the only argument. This program will acquire a read lock when the device is in a certain rotation ([0, 180]).

그런데 기기가 [0, 180] 범위에 있을 때 lock을 잡는다고만 되어 있지, 정확히 어떤 범위의 lock을 잡는 것인지 잘 이해가 안 가서 질문드립니다.

**Comment by lynring24** [comment]
range는 selector와 trial에서 적당히 설정해주시면 됩니다.

**Comment by lynring24** [comment]
Selector와 Trial에서 lock/unlock(degree, range)를 사용할 때,  range는 성능을 고려해 설정하시면 됩니다.

**Comment by hyojeonglee** [comment]
+selector 및 trial의 lock의 범위는 위 코멘트대로 범위에 따른 성능도 고려를 하면서, 테스트하고 싶은 상황에 따라 원하는 값을 직접 정해주시면 됩니다.

**Comment by kjp4155** [comment]
테스트 목적으로는 selector, trial 프로그램의 argument에 lock 범위를 받으면 좋을 것 같은데, 그렇게 해도 될까요?

**Comment by lynring24** [comment]
채점은 spec에 나와 있는 인터페이스(`./selector  INTEGER` 와 `./trial INTEGER` )를 기준으로 진행합니다.  조건에만 맞다면 문제가 없으며, 프로젝트 진행 과정에서 unit test로 구현하시는 것도 상관 없습니다.  

> 특정 환경에 대한 셋팅을 README.md에 정리하셨다면 참고 하겠습니다.

**Comment by hyojeonglee** [comment]
네 그 부분은 과제 스펙을 크게 거스르지 않으니 편하신대로 구현하시고 Readme에 테스트케이스별로 argument 명시해주시면 됩니다.
